resources/lib/oeWindows.py

```diff
@@ -141,7 +141,7 @@ def build_menu(self, struct, fltr=[], optional='0'):
                                 dictProperties['InfoText'] = oe._(setting['InfoText'])
                             if 'validate' in setting:
                                 dictProperties['validate'] = setting['validate']
-                            if 'values' in setting:
+                            if 'values' in setting and not setting['values'] is None:
```

Comment:
just a minor but would you mind changing the syntax a bit
```suggestion
                            if 'values' in setting and setting['values'] is not None:
```

Reply:
thanks, fixed :)